app/interactors/tasks/create.rb

```diff
@@ -2,7 +2,20 @@ module Tasks
   class Create
     include Interactor::Organizer
 
+    delegate :project, :task, to: :context
+
     organize Tasks::Creates::PrepareParams,
              Tasks::Save
+    after do
+      task = context.task
+      project = task.project
+      owner = project.users.find_by(project_memberships: { role: "owner" })
+      members = project.users.where.not(id: owner.id)
+
+      TaskMailer.task_created_owner(owner, task).deliver_later if owner.present?
+      members.each do |member|
+        TaskMailer.task_created_member(member, task).deliver_later
+      end
+    end
```

Comment:
You don't need to set here task and project, because you already has same when you delegating it higher
```suggestion
    after do
      owner = project.users.find_by(project_memberships: { role: "owner" })
      members = project.users.where.not(id: owner.id)

      TaskMailer.task_created_owner(owner, task).deliver_later if owner.present?
      members.each do |member|
        TaskMailer.task_created_member(member, task).deliver_later
      end
    end
```